backend/tests/teams.rs

```diff
@@ -181,3 +188,132 @@ async fn create_team_success() {
 
     assert_eq!(response.status(), 200);
 }
+
+async fn assert_team_size_setup() -> (DatabaseConnection, Uuid, Uuid) {
+    let database = Database::connect("sqlite::memory:").await.unwrap();
+
+    setup_schema(&database, team_invites::Entity).await;
+    setup_schema(&database, teams::Entity).await;
+    setup_schema(&database, users::Entity).await;
+    setup_schema(&database, email_confirmation::Entity).await;
+
+    let users = vec![
```

Comment:
Ordering, move this defintion of users just when you need it (so below team creation)